browser/brave_stats_updater.cc

```diff
@@ -142,6 +167,9 @@ void BraveStatsUpdater::OnSimpleLoaderComplete(
   if (!stats_updated_callback_.is_null())
     stats_updated_callback_.Run(final_url.spec());
 
+  // In case the first call was blocked by our timer.
+  (void) MaybeDoThresholdPing(0);
```

Comment:
why do you have `(void)` here?

Reply:
it returns a value. the return value is actually used in `brave_stats_tab_helper`. this is just calling that method back in case the score was >= `kMinimumUsageThreshold`, but was not triggered because the first usage ping hadn't fired yet. it's just making it clear that it's calling that back but not using the return value.

Reply:
are you getting a compiler error? You should normally only have a problem if the method has `WARN_UNUSED_RESULT` in which case you'd use ignore_result from base/macros.h. While using `(void)` here isn't technically wrong, it's just not standard practice in chromium code